Docker readme server (#246)
temporalio/docker-builds: docker-readme/server.md

`TEMPORAL_TLS_FRONTEND_SERVER_NAME` (defaults to unset)

This variable specifies which hostname your frontend should validate against when using TLS.

`TEMPORAL_TLS_CLIENT1_CA_CERT` (defaults to unset)

This variable specifies the TLS certificate that the frontend presents to external clients.

`TEMPORAL_TLS_CLIENT1_CA_CERT_DATA` (defaults to unset)

This variable allows you to pass a client security certificate as an object rather than a file path.

`TEMPORAL_TLS_CLIENT2_CA_CERT` (defaults to unset)

This variable allows you to specify an additional TLS certificate that the frontend presents to external clients.

`TEMPORAL_TLS_CLIENT2_CA_CERT_DATA` (defaults to unset)

This variable allows you to pass an additional client security certificate as an object rather than a file path.

`STATSD_ENDPOINT` (defaults to unset)

This variable allows you to specify an endpoint to connect to [statsd](https://github.com/statsd/statsd) for observability.

`PROMETHEUS_ENDPOINT` (defaults to unset)

This variable allows you to specify an endpoint to connect to Prometheus for observability.

`PROMETHEUS_TIMER_TYPE` (defaults to `histogram`)

> This variable specifies which [type of metrics](https://prometheus.io/docs/concepts/metric_types/) should be available at your Prometheus endpoint.

Reviewer: ```suggestion
This variable specifies which [type of Prometheus metric](https://prometheus.io/docs/concepts/metric_types/) is used for timer metrics.
```

`TEMPORAL_AUTH_AUTHORIZER` (defaults to unset)

This variable allows you to configure the [Authorizer](https://docs.temporal.io/self-hosted-guide/security#authorizer-plugin) plugin.

`TEMPORAL_AUTH_CLAIM_MAPPER` (defaults to unset)

This variable allows you to configure the [ClaimMapper](https://docs.temporal.io/self-hosted-guide/security#claim-mapper) plugin.

`TEMPORAL_JWT_KEY_SOURCE1` (defaults to unset)

This variable allows you to specify a JWT key source for a [ClaimMapper](https://docs.temporal.io/self-hosted-guide/security#default-jwt-claimmapper) integration.

`TEMPORAL_JWT_KEY_SOURCE2` (defaults to unset)

This variable allows you to specify another JWT key source for a ClaimMapper integration.

`TEMPORAL_JWT_KEY_REFRESH` (defaults to `1m`)

This variable specifies how frequently JWT keys should be refreshed.

`TEMPORAL_JWT_PERMISSIONS_CLAIM` (defaults to `permissions`)

> This vairable specifies the name of your JWT permissions claim.

Reviewer: ```suggestion
This variable specifies the property name of your JWT permissions claim.
```
